test/tests/functional/pbs_maintenance_reservations.py

```diff
@@ -425,7 +425,7 @@ def test_maintenance_reconfirm_reservation_and_run(self):
         self.server.manager(MGR_CMD_SET, SERVER,
                             {'managers': '%s@*' % TEST_USER})
         self.server.manager(MGR_CMD_SET, SERVER,
-                            {'scheduler_iteration': 3})
+                            {'scheduler_iteration': 3}, runas=TEST_USER)
```

Comment:
I'd rather you change the call to setting managers to (INCR, "%s@*" % TEST_USER) instead of running the second manager as TEST_USER.  I don't think the test needs to replace the managers to just TEST_USER.